presto-testing/src/main/java/io/prestosql/testing/AbstractTestingPrestoClient.java

```diff
@@ -130,6 +131,11 @@ private static ClientSession toClientSession(Session session, URI server, Durati
                 properties.put(connectorProperties.getKey() + "." + entry.getKey(), entry.getValue());
             }
         }
+        for (Entry<CatalogName, Map<String, String>> connectorProperties : session.getConnectorProperties().entrySet()) {
```

Comment:
Had to add this because I was running into a problem where the `allow_aggregation_pushdown` property was being dropped in the transaction block.

Reply:
is this change still needed?

Reply:
apparently it is not, dropped